src/ui/public/visualize/visualize.less

```diff
@@ -118,13 +118,9 @@ visualize-spy {
 
 .visualize-spy-container {
   flex: 1 1 auto;
-
   display: flex;
   flex-direction: column;
 
-  height: 482px;
```

Comment:
@cjcenizal @thomasneirynck 

I couldn't figure out why this rule existed and where such a specific number came from. I made some changes to the overflow / height states of the spy to make it actually usable within discovery. I checked that this doesn't effect the spy within dashboard and the visualize creation itself, but wanted to pass it by you just in case there's some part of Kibana I missed that actually requires this to exist.

![](https://d3vv6lp55qjaqc.cloudfront.net/items/1f1C1C2K2C3d2H3s053Q/Screen%20Recording%202017-06-08%20at%2011.30%20AM.gif?X-CloudApp-Visitor-Id=59773&v=4db1cc52)